buildifier/internal/factory.bzl

```diff
@@ -168,8 +168,6 @@ def buildifier_impl_factory(ctx, test_rule = False):
             fail("Cannot use 'no_sandbox' without a 'workspace'")
         workspace = ctx.file.workspace.path
 
-    out_file = ctx.actions.declare_file(ctx.label.name + ".bash")
```

Comment:
Note for the reviewer, this is already present in https://github.com/bazelbuild/buildtools/pull/1262/files#diff-c5e2ba5db338e88cf7c850afd25c044dfa97198fefa4b296370c9286ab81d415L184